docs/tools/new_tools_temp.md

```diff
@@ -78,67 +104,83 @@ To create a new Mbed OS project in a specified path:
     mbed-tools init -c <PATH>
     ```
 
-    <!--so how do I tell it where the existing copy is? In Mbed Studio you have to explicitly "tell it" where the local copy is-->
+<!--this seems to be an absolute path - had to start with `~/`. Can they work with relative paths? can they work with "here" to tell mbed-tools to work in the current directory?-->
 
-## List connected devices
 
-To list devices connected over USB:
+<!--If I can't symlink - if I have to copy my local version of Mbed OS - then this is less about savings space and more about allowing me to take a non-default version of Mbed OS, then?-->
+
+### Use an example application
+
+To create a local copy of an example application, use the `clone` command with the full GitHub URL listed below:
```

Comment:
You don't need to give the full github URL, you can use `mbedtools clone <example-name>` for ARMmbed examples.